Automattic/wp-calypso
# Login: Clear error notices once the user changes the username field

From #14173:

> In the case of an account that has not yet been activated, can we clear the notice once the user clears or updates the username field?

This PR handles this by firing a new action when the username field changes and clearing the error notice.

#### Testing instructions
  
1. Run `git checkout fix/clear-notice` and start your server
2. Createa a new unactive account from WPCC (start at akismet.com)
2. Open the [`Login` page](http://calypso.localhost:3000/log-in)
3. Try to log in with this account
4. Check that you see this message:
<img width="507" alt="screen shot 2017-07-12 at 17 30 40" src="https://user-images.githubusercontent.com/275961/28162797-40c6ad8a-67c0-11e7-9769-5c8a637a0118.png">
5. Change the username
6. Check that the error message is cleared
7. Try to log in with an account that has 2FA enabled
8. Enter an invalid code in the 2FA step, and check that an error shows
9. Change the value of the code
10. Check that the error is hidden.

  
#### Reviews
